src/App.js

```diff
@@ -1,24 +1,47 @@
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import foodsJson from './foods.json';
+import FoodBox from './components/FoodBox';
+import AddFoodForm from './components/AddFoodForm';
+import { Row } from 'antd';
+import Search from './components/Search';
 
 function App() {
+ const [food, setFoodList] = useState(foodsJson);
+  const [showFood, setShowFood] = useState(foodsJson);
+
+  const addFood = (newFood) => {
+    const updatedFood = [newFood, ...food];
+    setFoodList(updatedFood);
```

Comment:
Your `addFood()` function is not working because you're only updating `food` with `setFoodList` you need to update `showFood` as well, because that's the variable that you're using to loop through and display all the foods.
```
  const addFood = (newFood) => {
    const updatedFood = [newFood, ...food];
    setFoodList(updatedFood);
    setShowFood(updatedFood);
  };
```